shared/assets/Icons/IconCalendar/IconCalendar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "../icon.css";
+interface Props {
+  isContrastIconCalendar?: boolean;
+}
+
+export const IconCalendar = ({ isContrastIconCalendar }: Props) => {
+  const mode = isContrastIconCalendar ? "dark" : "light";
+  return (
+    <svg
+      width="16"
+      height="18"
```

Comment:
```suggestion
      className="w-4 h-[1.15rem]"
```
1. Не следует использовать width и height, когда есть className (в которые проще вставлять параметры)
2. Не следует использовать px